server/db_queries.js

```diff
@@ -0,0 +1,62 @@
+const mongoose = require("mongoose");
+const express = require("express");
+const router = express.Router();
+const conn_string =
+  "mongodb+srv://admin:passwordpassword@cluster0.wapen.mongodb.net/plusev?retryWrites=true&w=majority";
```

Comment:
We should probably read this string from a separate config file and have it included in the .gitignore. We can both just keep it locally ourselves. Otherwise anyone who looks at this repo would have access to our DB. 

It's probably not necessary but it's good practice and I guess there could be bots crawling for connections strings on public repos or something.